Python/test/coupons.py

```diff
@@ -34,17 +34,165 @@ def flat_rate(rate):
         2, CAL, ql.QuoteHandle(ql.SimpleQuote(rate)), ql.Actual365Fixed())
 
 
-def create_ibor_leg(ibor_idx, start, end):
+def create_ibor_leg(ibor_idx, start, end, payment_lag=0):
+    bdc = ibor_idx.businessDayConvention()
     sch = ql.MakeSchedule(effectiveDate=start,
                           terminationDate=end,
                           tenor=ibor_idx.tenor(),
-                          calendar=ibor_idx.fixingCalendar(),
-                          convention=ibor_idx.businessDayConvention(),
+                          calendar=CAL,
+                          convention=bdc,
+                          backwards=True)
+    return ql.IborLeg([1.0],
+                      sch,
+                      ibor_idx,
+                      ibor_idx.dayCounter(),
+                      bdc,
+                      [],
+                      [],
+                      [],
+                      [],
+                      [],
+                      False,
+                      ql.Period(),
+                      CAL,
+                      ql.Unadjusted,
+                      False,
+                      CAL,
+                      payment_lag)
```

Comment:
I would use keyword arguments for this — something like
```suggestion
    return ql.IborLeg([1.0],
                      sch,
                      ibor_idx,
                      ibor_idx.dayCounter(),
                      bdc,
                      exCouponCalendar=CAL,
                      paymentCalendar=CAL,
                      paymentLag=payment_lag)
```
We added them so we can avoid passing all the other defaulted args.  Same for the other calls below.

Reply:
Many thanks. I also added similar logic to payment calendar in `SubPeriodsLeg`, to make sure that all default args could be skipped, if needed.